src/app/feed/shared/connection-config.model.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import * as _ from "lodash";
+
+export interface IWanted {
+  audio?: boolean;
```

Comment:
In [this method](https://github.com/jangouts/jangouts/pull/147/files#diff-b5c6bf8f85bc96337664b5bdf08b7ac6R100) the requested attribute is set as an empty object, so audio and video need to be optional in the interface. I will put a TODO comment to review this code, but by now I want to keep moving with the migration.